### PR TITLE
compilation bugs fixed

### PR DESCRIPTION
**Pull request by raphaelMuth**
Sorry for the late time.
There is a fork updated to typescript 2.6.
For me it stops in "loading assets"

### CODE REVIEW

File: src/Settings.ts [hunk 1]
```diff
@@ -78,12 +78,15 @@ module Settings
     }
 
     export function getUrlVars()
-    {
+	{
+		//TODO
+		//VERIFICAR O RETORNO QUE PODE ESTAR INCORRETO PARA O METODO
```
Comment (CiaranMcCann) [inline src/Settings.ts:83]: ?

File: src/Settings.ts [hunk 2]
```diff
@@ -78,12 +78,15 @@ module Settings
...
         var vars = {};
         window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function (m, key, value) 
         {
-            vars[key] = value;
-            return true;
+			vars[key] = value;
+			//return true;
+            return "";
```
Comment (CiaranMcCann) [inline src/Settings.ts:89]: Whats this change about?

File: src/networking/Server.ts [hunk 3]
```diff
@@ -22,11 +22,11 @@ declare var Util;
...
-    var Events = require('./Events');
-    var ServerUtilies = require('./ServerUtilies');
-    var GameLobby = require('./GameLobby');
-    var ServerSettings = require('./ServerSettings');
-    var Lobby = require('./Lobby');
+    //var Events = require('./Events');
+    //var ServerUtilies = require('./ServerUtilies');
+    //var GameLobby = require('./GameLobby');
+    //var ServerSettings = require('./ServerSettings');
+    //var Lobby = require('./Lobby');
```
Comment (CiaranMcCann) [inline src/networking/Server.ts:29]: ?

**Comment by CiaranMcCann** [comment]
@raphaelMuth Does this now compile? Could you post the console output.

File: src/Settings.ts (hunk 1 above)
Comment (raphaelMuth) [reply src/Settings.ts:83]: I'm brazilian, that's why the spelling. 
The changes initaly was for me to test.
I let that "TODO" to check if the return "" would still make the method work.
I forgot to remove it

File: src/Settings.ts (hunk 2 above)
Comment (raphaelMuth) [reply src/Settings.ts:89]: Visual studio says the method could not return true just a string, then i setted it to return "", and check later.
I checked it in the chrome console now, and it works perfectly to get the vars after ? or & in url

File: src/networking/Server.ts (hunk 3 above)
Comment (raphaelMuth) [reply src/networking/Server.ts:29]: Visual studio was giving the error "Duplicate name" for those guys

**Comment by raphaelMuth** [comment]
It's in portuguese, sorry
1>------ Recompilação total iniciada: Projeto: WormsHTML5, Configuração: Release Any CPU ------
1>  WormsHTML5 -> C:\Users\Raphael\Documents\Visual Studio 2017\Worms-Armageddon-HTML5-Clone-master\bin\WormsHTML5.dll
========== Recompilar Tudo: 1 com êxito, 0 com falha, 0 ignorados ==========

VS 2017

**Comment by raphaelMuth** [comment]
Sorry, i will check this guy a little more, and update the PR
Some wild errors appeared, i was looking in the wrong place

**Comment by raphaelMuth** [comment]
I'm going exclude this PR, update it from the start again, take more attention and time.
At least can you send bandwithmonitor.ts ? its missing in master

**Pull request closed without merging**